src/views/notifications/Create.vue

```diff
@@ -0,0 +1,323 @@
+                              User
+                            </label>
+                            <TailSelect
+                              v-model="this.create_notification.target_id"
+                              :class="'form-control' + (this.validation_error?.target_id != null ? ' border-theme-6' : '')"
+                              :options="{
+                                  search: true,
+                                  classNames: 'w-full'
+                              }"
+                            >
+                              <option :value="user.id" v-for="user in this.users" v-bind:key="user.id" :selected="this.create_notification.target_id === user.id">
+                                {{ user.name }}
+                              </option>
+                            </TailSelect>
+                            <div v-if="parseInt(this.create_notification.type) !== 2" class="text-theme-6 mt-2 mb-4">
+                              Icon Type is selected!
+                            </div>
+                            <div v-if="this.validation_error?.target_id != null" class="text-theme-6 mt-2 mb-4">
+                              {{ this.validation_error?.type[0] }}
+                            </div>
+                          </div>
+                        </div>
+                        <button class="btn btn-primary w-20" @click="createNotification(this.create_notification)">
+                          Create
+                        </button>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <!-- END: Notification Creation Tab -->
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { defineComponent } from 'vue'
+import Header from './Components/Header'
+import axios from 'axios'
+import DefaultTab from './Components/DefaultTab'
+import { useToast } from 'vue-toastification'
+const toast = useToast()
+
+export default defineComponent({
+  components: {
+    DefaultTab,
+    Header
+  },
+  data() {
+    return {
+      active_tab: false,
+      users: [],
+      validation_error: {},
+      search: {
+        user: ''
+      },
+      create_notification: {
+        title: 'New Notification',
+        content: 'New Content',
+        color: '#000000',
+        type: 1,
+        target_id: null,
+        icon: 'BellIcon',
+        user_id: 0
+      }
+    }
+  },
+  mounted() {
+    this.fetchUsers('users?paginate=0')
+  },
+  methods: {
+    fetchUsers(page) {
+      const loader = this.$loading.show()
+      axios.get(page)
+        .then(response => {
+          this.users = response.data
+          loader.hide()
+        })
+        .catch(error => {
+          this.validation_error = error.response.data.data.errors
+          toast.error(error.response.data.message)
+          console.error(error)
```

Comment:
Remove debug messages if not required constantly

Reply:
Solved in https://github.com/KrbDevelopment/Articly-Frontend/commit/a36b1eac17ffc2d9a45464809cd1bf97e2f9d2d6